Control/lib/api.js

```diff
@@ -70,6 +73,10 @@ module.exports.setup = (http, ws) => {
   ctrlProxy.methods.forEach(
     (method) => http.post(`/${method}`, coreMiddleware, (req, res) => ctrlService.executeCommand(req, res))
   );
+  http.post('/NewEnvironmentRequest', coreMiddleware, (req, res) => aliecsReqHandler.add(req, res));
+  http.post('/GetEnvironmentRequests', coreMiddleware, (req, res) => aliecsReqHandler.getAll(req, res));
```

Comment:
Is there a reason for which we keep the gRPC naming? If not, I would not continue to follow gRPC convention for our own request. I would suggest that we follow the NodeJS standard similar to below:
```
http.post('environment')
http.get('environments')
http.delete('environment/:id')
```
I believe our frontend loader does not support 'delete' so for now for delete we keep
`http.post('environment/:id')`